lib/cext/preprocess.rb

```diff
@@ -246,6 +246,14 @@ def read_write_field(struct_var_name, field_name, leaking)
       {
         match: /if \(stmt_wrapper->refcount == 0\) {/,
         replacement: MYSQL2_FREE_STATEMENT
+      },
+      { 
+        match: /\bparams_enc\[i\](\s*(?:[\),;]|==|!=))/,
```

Comment:
I'm a bit concerned with the duplication here for `(?:[\),;]|==|!=)`, maybe we could just extract that part in a `NO_ASSIGNMENT` constant?

Reply:
Fixed, extracted to constant.